diff -ruN build.old/static/service-worker.js build/static/service-worker.js
--- build.old/static/service-worker.js	2020-03-14 18:33:58.231000000 +0100
+++ build/static/service-worker.js	2020-03-14 18:46:05.988012240 +0100
@@ -34,6 +34,6 @@
 workbox.precaching.precacheAndRoute(self.__precacheManifest, {});
 
 workbox.routing.registerNavigationRoute(workbox.precaching.getCacheKeyForURL("/user-interface/classification/index.html"), {
-  
-  blacklist: [/^\/_/,/\/[^/?]+\.[^/]+$/],
+
+  blacklist: [/^\/_/,/\/[^/?]+\.[^/]+$/, /^\/user-interface\/classification\/admin\//, /^\/user-interface\/classification\/auth\//, /^\/user-interface\/classification\/images\//],
 });
